migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -1,25 +1,18 @@
 [id="planning-migration-3-4"]
-= Planning your migration
+= Differences between {product-title} 3 and 4
 include::modules/common-attributes.adoc[]
 :context: planning-migration-3-4
 
 toc::[]
 
-Before performing your migration to {product-title} {product-version}, it is important to take the time to properly plan for the transition. {product-title} 4 introduces architectural changes and enhancements, so the procedures that you used to manage your {product-title} 3 cluster might not apply for {product-title} 4.
-
-[NOTE]
-====
-This planning document assumes that you are transitioning from {product-title} 3.11 to {product-title} {product-version}.
-====
+{product-title} {product-version} introduces architectural changes and enhancements, so the procedures that you used to manage your {product-title} 3 cluster might not apply to {product-title} 4.
 
 ifndef::openshift-origin[]
-This document provides high-level information on the most important xref:../migrating_from_ocp_3_to_4/planning-migration-3-4.adoc#migration-comparing-ocp-3-4[differences between {product-title} 3 and {product-title} 4] and the most noteworthy xref:../migrating_from_ocp_3_to_4/planning-migration-3-4.adoc#migration-considerations[migration considerations]. For detailed information on configuring your {product-title} 4 cluster, review the appropriate sections of the {product-title} documentation. For detailed information on new features and other notable technical changes, review the xref:../release_notes/ocp-4-8-release-notes.adoc#ocp-4-8-release-notes[OpenShift Container Platform 4.8 release notes].
+For detailed information on configuring your {product-title} 4 cluster, review the appropriate sections of the {product-title} documentation. For detailed information on new features and other notable technical changes, review the xref:../release_notes/ocp-4-8-release-notes.adoc#ocp-4-8-release-notes[OpenShift Container Platform 4.8 release notes].
```

Comment:
Remove the word detailed ahead of information in both instances for minimalism purposes.